using_images/xpaas_images/fuse.adoc

```diff
@@ -65,23 +63,32 @@ The Maven Archetype catalog includes the following examples:
 
 Begin by selecting the archetype which matches the type of application you would like to create.
 
-==== Simple Workflow to Create an Application from Maven Archetype Catalog
+=== Create an Application from the Maven Archetype Catalog
 
-. Use the Maven archetype catalog to create a sample project with the required resources. In order to use archetypes it is important to configure the Maven repositories which holds the archetypes and artifacts you may need:
+You must configure the Maven repositories, which holds the archetypes and artifacts you may need, before creating a sample project:
 
 * JBoss Fuse repository: `https://repo.fusesource.com/nexus/content/groups/public/`
 * RedHat GA repository: `https://maven.repository.redhat.com/ga`
-+
-You can then use the Maven archetype catalog to create a sample project with the required resources. For example, the following command will create a sample project:
+
+[start=1]
+. Use the maven archetype catalog to create a sample project with the required resources. The command to create a sample project is:
+
 +
 ----
 $ mvn archetype:generate \
   -DarchetypeCatalog=https://repo.fusesource.com/nexus/content/groups/public/archetype-catalog.xml \
-  -Dfilter=io.fabric8.archetypes:
+  -DarchetypeGroupId=io.fabric8.archetypes \
+  -DarchetypeVersion=2.2.0.redhat-079 \
+  -DarchetypeArtifactId=[archetype-name]
 ----
+[NOTE]
+Replace [archetype-name] with the name of the archetype that you want to use. For example, karaf-camel-log-archetype will create a new Camel log example.
```

Comment:
Should have some ===='s around the box, as per:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions

Also, we try not to use future tense, so maybe:

"For example, karaf-camel-log-archtype creates a new Camel log example."

Also, should "karaf-camel-log-archtype" have markup of some kind? If it's a service, maybe *'s?